Sources/AppStoreConnectCLI/Services/Operations/ListBetaGroupsOperation.swift

```diff
@@ -0,0 +1,45 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import AppStoreConnect_Swift_SDK
+import Combine
+import Foundation
+
+struct ListBetaGroupsOperation: APIOperation {
+    struct ListBetaGroupsDependencies {
+        let betaGroups: (APIEndpoint<BetaGroupsResponse>) -> Future<BetaGroupsResponse, Error>
+    }
+
+    private let endpoint: APIEndpoint<BetaGroupsResponse>
+
+    init(options: ListBetaGroupsOptions) {
+        endpoint = .betaGroups(include: [.app])
+    }
+
+    func execute(
+        with dependencies: ListBetaGroupsDependencies
+    ) -> AnyPublisher<[BetaGroup], Error> {
+        let response = dependencies.betaGroups(endpoint)
+
+        let betaGroups = response.map { response -> [BetaGroup] in
+            var betaGroups: [String: BetaGroup] = [:]
```

Comment:
```suggestion
            var betaGroups: [String: [BetaGroup]] = [:]
```
I think an appId may have multiple betaGroups?

Reply:
You might be right but I'm about to rework how this works because of filters and such